_includes/_solo_boinc_intro.htm

```diff
@@ -4,12 +4,12 @@
     <div class="container">
         <div class="row">
             <div class="col-xs-12 col-sm-8">
-                <h5>Not interested in earning BOINC rewards?</h5>
+                <h5 class="text-bold">Don't Want BOINC rewards?</h5>
                 <p>Switch to the <a href="/Guides/investor-gridcoin-setup.htm">Investor guide</a>. You do not need to install or run BOINC in order to use GRC as a cryptocurrency.</p>
                 <h4 class="text-bold">BOINC Introduction</h4>
-                <p>In order to earn more Gridcoin, you need to crunch one or more whitelisted <a href="https://boinc.berkeley.edu/">BOINC</a> projects within team Gridcoin. You must install the <a href="https://boinc.berkeley.edu/">BOINC</a> client, select several <a href="https://boinc.berkeley.edu/">BOINC</a> projects and get crunching!</p>
-                <p>To retrieve your crunching rewards by solo mining, you must have a balance of Gridcoin sufficient to stake before 180 days elapses. To be safe you should have enough Gridcoin to stake approximately once every month, this would be around 5,000 GRC.</p>
-                <p>GRC can be acquired in a number of different ways <a href="/exchange.htm">here.</a> You can also start out by <a href="/Guides/pool.htm">pool mining</a>, which requires no initial investment of Gridcoin to retrieve your rewards.</p>
+                <p>To earn Gridcoin, you need to crunch whitelisted <a href="https://boinc.berkeley.edu/">BOINC</a> projects on team Gridcoin. You must install the BOINC client, select several BOINC projects</p>
```

Comment:
>  You must install the BOINC client, select several BOINC projects

Should be ' You must install the BOINC client and select several BOINC projects' 
No need for a comma if there are only 2 things being stated. Also no fullstop at the end of the sentence.

Reply:
Think I was taking from a list and didn't catch that, will fix